charts/rancher-logging/105.0.0-rc1+up4.8.0/values.yaml

```diff
@@ -6,7 +6,7 @@ replicaCount: 1
 
 image:
   repository: rancher/mirrored-kube-logging-logging-operator
-  tag: 4.8.0
```

Comment:
The linked CVE is for `4.4.0` are those CVEs not fixed yet in `4.8.0`? To fix issues in `4.4.0` you'll probably want to be targeting 2.9 / 2.8

Reply:
That's right the linked CVE is for 4.4.0. The idea here is, since the fix will be needed for 2.9, why not follow upstream and update to the latest used version in both branches?

Reply:
I think if we do that, then we also need to bump the charts too.

Per my analysis here: https://github.com/rancher/image-mirror/pull/706#pullrequestreview-2261719867
(assuming we agree with that logic still)

Then we would want to use the following logic:

1. Always prefer to match the version of images used in upstream charts,
  -  Crucial because per that bug the other day, if we allow image versions to drift between each other then bugs happen.
3. When we determine upstream lags for no reason, update local and PR to upstream secondary priority.

Reply:
I actually made a mistake when interpreting the values from upstream, let's keep that value as it is then!